sdk/core/core-http/lib/policies/tracingPolicy.ts

```diff
@@ -11,17 +12,28 @@ import {
 import { WebResource } from "../webResource";
 import { HttpOperationResponse } from "../httpOperationResponse";
 
-export function tracingPolicy(): RequestPolicyFactory {
+export interface TracingPolicyOptions {
+  userAgent?: string;
+}
+
+export function tracingPolicy(policyOptions: TracingPolicyOptions = {}): RequestPolicyFactory {
   return {
     create(nextPolicy: RequestPolicy, options: RequestPolicyOptions) {
-      return new TracingPolicy(nextPolicy, options);
+      return new TracingPolicy(nextPolicy, options, policyOptions);
```

Comment:
Now that there are two options in scope do you mind renaming them to differentiate them a bit?

Reply:
Good point, renaming the latter to be less confusing.